manager/subscriptionmanager/subscriptionmanager.py

```diff
@@ -139,19 +140,61 @@ def sendVirtGuests(self, guests):
 
     def hypervisorCheckIn(self, config, mapping, type=None):
         """ Send hosts to guests mapping to subscription manager. """
-
         serialized_mapping = {}
-        for host, guests in mapping.items():
-            serialized_mapping[host] = [guest.toDict() for guest in guests]
-        self.logger.info("Sending update in hosts-to-guests mapping: %s" % json.dumps(serialized_mapping, indent=4, sort_keys=True))
 
         self._connect(config)
+        self.logger.debug("Checking if server has capability 'hypervisor_async'")
+        is_async = self.connection.has_capability('hypervisors_async')
```

Comment:
Stay backwards compatible with older versions of rhsm, for example:
    is_async = hasattr(self.connection, 'has_capability') and self.connection.has_capability('hypervisors_async')